### DISCUSSION TITLE
Container App Secrets to env variable missing

### DISCUSSION THREAD

**Discussion by michaell104** [Azure/ResourceModules#2995] [Q&A]
The container app module seems to be missing the ability to pass the secrets from the container app to the individual container.
Currently I can pass a list to 
````
@description('Optional. The secrets of the Container App.')
@secure()
param secrets object = {}`
````
Which is referenced here 
````
var secretList = !empty(secrets) ? secrets.secureList : []
````
The `secretList` is then referenced at the container app level with `secrets: secretList` but I should then be able to pass these to the `template: env:` of the individual container from the container app secrets.

**Comment by eriqua** [comment]
Hi @cwp-michaell, thanks for opening this discussion and sorry for the late reply. Discussions are currently not monitored promptly and are planned to be disabled in favor of the new issue template [Questions/Feedback].

I'm converting this discussion to an issue in order for the team to be able to triage it.

Please refer to issue #3140 going forward.
